src/bin/lib/subcommands.c/runprogram.h

```diff
@@ -183,6 +183,15 @@ execute_subprogram(Program *prog)
 	int outpipe[2] = { 0, 0 };
 	int errpipe[2] = { 0, 0 };
 
+	/* first level sanity check */
+	if (access(prog->program, F_OK | X_OK) == -1)
+	{
+		fprintf(stderr, "Failed to find executable program at \"%s\": %s",
+				prog->program,
+				strerror(errno));
+		return;
```

Comment:
Do `prog->returnCode` and `prog->error` need to be set in these code paths?

Reply:
Good point, thanks! Done now.